# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../chntpw/hotfix.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- chntpw-080526/Makefile.vanilla	2009-10-28 19:26:05.000000000 +0100
+++ chntpw-080526/Makefile	2009-10-28 19:26:41.000000000 +0100
@@ -13,7 +13,7 @@
 CC=gcc
 
 # Force 32 bit
-CFLAGS= -DUSEOPENSSL -g -I. -I$(OSSLINC) -Wall -m32
+CFLAGS= -DUSEOPENSSL -g -I. -I$(OSSLINC) -Wall
 OSSLLIB=$(OSSLPATH)/lib
 
 # 64 bit if default for compiler setup
@@ -22,7 +22,7 @@
 
 
 # This is to link with whatever we have, SSL crypto lib we put in static
-LIBS=-L$(OSSLLIB) $(OSSLLIB)/libcrypto.a
+LIBS=-L$(OSSLLIB) -lcrypto
 
 all: chntpw chntpw.static cpnt reged reged.static
 
